### PR TITLE
fetch from redis

### PR DESCRIPTION
**Pull request by NourAlharithi**

### CODE REVIEW

File: src/dlob-subscriber/DLOBSubscriberIO.ts [hunk 1]
```diff
@@ -124,13 +124,39 @@ export class DLOBSubscriberIO extends DLOBSubscriber {
...
+
 		this.redisClient.client.publish(
 			`orderbook_${marketType}_${l2Args.marketIndex}`,
 			JSON.stringify(l2Formatted)
 		);
 		this.redisClient.client.set(
 			`last_update_orderbook_${marketType}_${l2Args.marketIndex}`,
-			JSON.stringify(l2Formatted)
+			JSON.stringify(l2Formatted_depth100)
+		);
```
Comment (wphan) [inline src/dlob-subscriber/DLOBSubscriberIO.ts:148]: maybe have the key default to `_depth_100` when getting so we dont store the same value in 2 keys?

File: src/index.ts [hunk 2]
```diff
@@ -742,6 +755,96 @@ const main = async () => {
...
+					if (
+						redisL2 &&
+						Math.abs(
+							parseInt(JSON.parse(redisL2).slot) - slotSource.getSlot()
+						) < 10
+					)
+						l2Formatted = redisL2;
+				}
+
+				if (l2Formatted) {
```
Comment (wphan) [inline src/index.ts:841]: i think it'd be good to introduce a metric here to track cache hit/misses

File: src/dlob-subscriber/DLOBSubscriberIO.ts (hunk 1 above)
Comment (wphan) [reply src/dlob-subscriber/DLOBSubscriberIO.ts:148]: actually it doesnt look like dlob server even reads cache for no depth param, so maybe it's ok to just remove this

Comment (NourAlharithi) [reply src/dlob-subscriber/DLOBSubscriberIO.ts:148]: oh this is there because of the websockets, which do the entire depth and get this value

**Pull request merged**